src/components/route/addBook.jsx

```diff
@@ -22,15 +24,25 @@ const AddBook = () => {
   };
 
   return (
-    <div className="book-form">
-      <div>
-        <h3>ADD NEW BOOK</h3>
+    <div className="FormContianer-all">
+      <div className="FormContianer">
+        <div className="title">
+          <h3>ADD NEW BOOK</h3>
+        </div>
+
+        <form className="form">
+          <div className="input-container">
+            <input value={title} type="text" id="title" placeholder="Book Title Here..." onChange={(e) => setTitle(e.target.value)} />
+          </div>
+          <div className="input-container">
+            <input value={author} type="text" id="name" placeholder="Author Name Here..." onChange={(e) => setAuthor(e.target.value)} />
+          </div>
+          <div className="input-container">
+            <button type="button" value="AddNew" onClick={handleSubmit}> ADD BOOK </button>
+          </div>
+        </form>
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirement as your previous code reviewer also asked for the changes related to the design. As per the requirement, your design should match exactly to Zeplin's design for the bookStore CMS project but you didn't implement any changes. Building the exact clone is proof of your design skills, makes you detail-oriented and helps you to be a better developer. Your design doesn't contain the user icon as mentioned previously and your `add book` category drop-down menu is missing.

### Requirement
![image](https://github.com/misikir21/bookstore/assets/66476329/f2df6c8e-4c9c-4fd0-b93b-ebd313a2f933)

### Design Comparison
<table>
<tr>
<th>Zeplin Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-04 14-58-52](https://user-images.githubusercontent.com/63972670/268999721-ff833b79-e13d-49fd-a035-b941c0c0793b.PNG)

</td>
<td>

![Screenshot from 2023-05-04 14-59-01](https://user-images.githubusercontent.com/63972670/268999857-1bc02c6d-0a2b-4081-8b42-a880764f1624.PNG)

</td>
</tr>
</table>